packages/node_modules/@ciscospark/react-component-space-item/src/__snapshots__/index.test.js.snap

```diff
@@ -1,5 +1,59 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`SpaceItem component correctly renders an active space 1`] = `
+<div
+  className="space-item item active"
```

Comment:
Since the snapshot file has the `active` class, we don't need to verify it in the actual test.